### PR TITLE
Update eip-725 to adhere to the last definition

### PR DESCRIPTION
**Pull request by frozeman**
This PR adjust 725 to the latest discussion version found here: https://github.com/ethereum/EIPs/issues/725

### CODE REVIEW

File: EIPS/eip-725.md [hunk 1]
```diff
@@ -1,55 +1,93 @@
----
+```
 eip: 725
-title: Proxy Account
-author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
+title: ERC-725 Smart Contract Based Account
+author: Fabian Vogelsteller <fabian@lukso.network>, Tyler Yasaka (@tyleryasaka)
```
Comment (MicahZoltu) [inline EIPS/eip-725.md:4]: If you are the primary author, I recommend leaving your GitHub username in here as the bot struggles with email addresses sometimes.  If you want both you can do:
```suggestion
author: Fabian Vogelsteller (@frozeman), Fabian Vogelsteller <fabian@lukso.network>, Tyler Yasaka (@tyleryasaka)
```

File: EIPS/eip-725.md [hunk 2]
```diff
@@ -1,55 +1,93 @@
...
 discussions-to: https://github.com/ethereum/EIPs/issues/725
 status: Draft
 type: Standards Track
 category: ERC
+requires: ERC165, ERC173, ERC1271 (optional)
```
Comment (MicahZoltu) [inline EIPS/eip-725.md:9]:
```suggestion
requires: 165, 173, 1271
```
Frontmatter is *very* strict on formatting.

File: EIPS/eip-725.md [hunk 3]
```diff
@@ -1,55 +1,93 @@
...
 created: 2017-10-02
----
+updated: 2020-07-02
```
Comment (MicahZoltu) [inline EIPS/eip-725.md:11]:
```suggestion
```
There is no `updated` field.  git commit history is the source of truth for that information should someone want it.

File: EIPS/eip-725.md [hunk 4]
```diff
@@ -1,55 +1,93 @@
----
+```
```
Comment (MicahZoltu) [inline EIPS/eip-725.md:1]:
```suggestion
---
```
`---` was correct, this is not a code block, it is markdown frontmatter (YML encoded internally).

File: EIPS/eip-725.md [hunk 5]
```diff
@@ -1,55 +1,93 @@
...
+author: Fabian Vogelsteller <fabian@lukso.network>, Tyler Yasaka (@tyleryasaka)
 discussions-to: https://github.com/ethereum/EIPs/issues/725
 status: Draft
 type: Standards Track
 category: ERC
+requires: ERC165, ERC173, ERC1271 (optional)
 created: 2017-10-02
----
+updated: 2020-07-02
+```
```
Comment (MicahZoltu) [inline EIPS/eip-725.md:12]:
```suggestion
---
```

File: EIPS/eip-725.md [hunk 6]
```diff
@@ -1,55 +1,93 @@
...
+
+**This is the new 725 v2 standard, that is radically different from [ERC 725 v1](https://github.com/ethereum/EIPs/blob/ede8c26a77eb1ac8fa2d01d8743a8cf259d8d45b/EIPS/eip-725.md). ERC 725 v1 is be moved to #734 as a new key manager standard.**
+--------
```
Comment (MicahZoltu) [inline EIPS/eip-725.md:15]:
```suggestion
```
EIPs should be written for future readers who don't care about the history.  If someone wants to see the progression of an EIP, they can go read through git history or the discussion link.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
This feels like a totally new EIP, in which case I would recommend a new EIP number.  That being said, I am kind of annoyed by people treating draft EIPs as final so I'm not actually opposed to this since maybe this kind of drastic rewrite will help discourage that behavior.

The frontmatter stuff does need to be fixed prior to merge though.

File: EIPS/eip-725.md (hunk 2 above)
Comment (DONALDSULLENDER) [reply EIPS/eip-725.md:9]: donaldsullender@gmail.com

File: EIPS/eip-725.md (hunk 1 above)
Comment (frozeman) [reply EIPS/eip-725.md:4]: good point

File: EIPS/eip-725.md (hunk 2 above)
Comment (frozeman) [reply EIPS/eip-725.md:9]: I think having the ERC is important as there will be other standards proposals from other chains, that are relevant to Ethereum smart contracts as well. In LUKSO we use `LSP` (LUKSO standard proposals), that will heavily extend 725

File: EIPS/eip-725.md (hunk 3 above)
Comment (frozeman) [reply EIPS/eip-725.md:11]: ok

File: EIPS/eip-725.md (hunk 6 above)
Comment (frozeman) [reply EIPS/eip-725.md]: removed

File: EIPS/eip-725.md (hunk 2 above)
Comment (MicahZoltu) [reply EIPS/eip-725.md:9]: If you feel strongly about this, I recommend bringing it up to the EIP editors in Discord or via a pull request to EIP-1 and the validator bot.  In the meantime, I recommend making this suggested change if you want to get this merged.

File: EIPS/eip-725.md [hunk 7]
```diff
@@ -1,55 +1,90 @@
 ---
 eip: 725
-title: Proxy Account
-author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
+title: ERC-725 Smart Contract Based Account
+author: Fabian Vogelsteller <@frozeman>, Tyler Yasaka (@tyleryasaka)
```
Comment (MicahZoltu) [inline EIPS/eip-725.md:4]:
```suggestion
author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
```
GitHub usernames are `()` and email addresses are `<>`.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
Once fixed, CI should pass and we can get this merged.

**Pull request merged**